src/core/org/apache/jmeter/threads/JMeterThread.java

```diff
@@ -148,7 +150,7 @@
 
     private final ReentrantLock interruptLock = new ReentrantLock(); // ensure that interrupt cannot overlap with shutdown
 
-    public JMeterThread(HashTree test, JMeterThreadMonitor monitor, ListenerNotifier note) {
+    public JMeterThread(HashTree test, JMeterThreadMonitor monitor, ListenerNotifier note,Boolean isSameUser) {
```

Comment:
Parameter naming is kind of hard to understand.
If you add a parameter to existing constructor/method, you might want to just add another constructor and delegate. That would prevent adding `,false)` all over the place

Reply:
Agreed